content/articles/searching-and-sorting-techniques/index.md

```diff
@@ -0,0 +1,230 @@
+### Introduction

```

Comment:
Please add Front matter to begin with.

Reply:
The contributor does not add the front matter